### ISSUE TITLE
Footer padding consistency

### ISSUE THREAD

**Issue by dsegovia90** [sego90/slack-timezoner#20]
There is an inconsistency on the padding of the inner element of the footer.

![image](https://user-images.githubusercontent.com/14206991/29140402-f0620a14-7d0f-11e7-9619-ddc34fda3569.png)

**Comment by jdrichardstech** [comment]
i am not getting the footer issue on my computer in chrome, firefox, or safari or with my chrome window resizer extension. what do you use to check mobile? is there a better tool to use for testing?

**Comment by jdrichardstech** [comment]
check footer-issue branch and let me know if this solved the issue

**Comment by dsegovia90** [comment]
I modified it. There is a way to check the pixels in the padding and margin in the chrome dev tools. Control+Shift+C select the div and at the very bottom of the right panel it shows the pixels. Attaching screenshot (look where the pointer is). In the screenshot it's fixed. 65px in the top and 65px on the bottom. My guess is that I explained poorly the issue. lol
![image](https://user-images.githubusercontent.com/14206991/29200239-2d5b1852-7e19-11e7-9240-4d995b9209d4.png)
